tests/caasp/stack_initialize.pm

```diff
@@ -35,6 +35,10 @@ sub run {
     # Update kubectl
     assert_script_sudo "zypper -n up kubernetes-client", 300;
 
+    # Install Helm
+    assert_script_sudo "zypper ar --refresh --no-gpgcheck https://download.opensuse.org/repositories/devel:/CaaSP:/2.0:/ControllerNode/SLE_12_SP3/devel:CaaSP:2.0:ControllerNode.repo";
```

Comment:
I am just guessing here, but I am afraid of `devel` and `2.0` in url - does this work on SP3?
Don't we have more reliable source for SP3?
What about `https://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12-SP3/standard/`

Reply:
The thing is that in this repository that you mentioned we have a new version of Helm, which is incompatible with the version of Tiller we ship in CaaSP.

```bash
tux@ultron:~> helm install stable/redis
Error: incompatible versions client[v2.7.2] server[v2.6.1]
```

Reply:
I see, then ignore this comment